.github/workflows/build.yml

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2021 Valve Corporation
+# Copyright (c) 2021 LunarG, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Lenny Komow <lenny@lunarg.com>
+
+name: CI Build
+
+on:
+    push:
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    linux:
+        runs-on: ubuntu-18.04
```

Comment:
We officially support both Ubuntu 18.04 and Ubuntu 20.04.  Can we get `ubuntu-20.04` in too?

Reply:
Sure, what's an extra 4 builds? I'll add that.